extensions/main.tf

```diff
@@ -90,6 +90,8 @@ module "monitoring" {
 }
 
 resource "aws_ssm_parameter" "addons" {
+  # checkov:skip=CKV_AWS_337: Used by copilot potential changes to platform-helper before CMK can be enabled
```

Comment:
I don't quite understand what this one means

Reply:
This one is an extension/learning from DBTP-1128. The ECS container has issues decrypting items in KMS that are encrypted by a CMK without changes to the commandline used by copilot. As the commandline is defined in platform-helper that means we need to update it to change the command line. Hope that makes more sense!

It's another one we should probably look to raise it's own ticket for

Reply:
Changed comment to hopefully clarify

Reply:
Makes sense and agree it sounds like a separate ticket